hosts.txt

```diff
@@ -15,130 +16,290 @@
 
-# [celtra.com]
-127.0.0.1 cdn.celtra.com
+# [Adcolony ads www.adcolony.com]
+127.0.0.1 adc3-launch.adcolony.com
+127.0.0.1 ads30.adcolony.com
+127.0.0.1 androidads20.adcolony.com
+127.0.0.1 events3.adcolony.com
+127.0.0.1 events3alt.adcolony.com
+127.0.0.1 wd.adcolony.com
+
+# [appads appadspromotions.com/ ?]
+127.0.0.1 jupiter.appads.com
+127.0.0.1 jupiter1.appads.com
+127.0.0.1 jupiter2.appads.com
+127.0.0.1 jupiter3.appads.com
+127.0.0.1 neptune.appads.com
+127.0.0.1 neptune1.appads.com
+127.0.0.1 neptune2.appads.com
+127.0.0.1 neptune3.appads.com
+127.0.0.1 req.appads.com
+127.0.0.1 req1.appads.com
+127.0.0.1 req2.appads.com
+127.0.0.1 req3.appads.com
+127.0.0.1 saturn.appads.com
+127.0.0.1 saturn1.appads.com
+127.0.0.1 saturn2.appads.com
+127.0.0.1 saturn3.appads.com
+
+# [applovin]
+127.0.0.1 a.applovin.com
+127.0.0.1 d.applovin.com
+127.0.0.1 pdn.applovin.com
+127.0.0.1 rt.applovin.com
+
+# [apptimize]
+127.0.0.1 brahe.apptimize.com
+127.0.0.1 md-a-c.apptimize.com
+127.0.0.1 md-a-s.apptimize.com
+
+# [BuzzCity mobile advertising www.buzzcity.com]
+127.0.0.1 apps.buzzcity.net
+127.0.0.1 click.buzzcity.net
+127.0.0.1 show.buzzcity.net
+
+# [Celtra mobile advertising www.celtra.com/home]
 127.0.0.1 ads.celtra.com
-127.0.0.1 cache-ssl.celtra.com
 127.0.0.1 cache.celtra.com
+127.0.0.1 cache-ssl.celtra.com
+127.0.0.1 cdn.celtra.com
 127.0.0.1 track.celtra.com
 
-# [inner-active.mobi]
-127.0.0.1 wv.inner-active.mobi
+# [misc cloudfront hosted ads]
+127.0.0.1 d25xkbr68qqtcn.cloudfront.net
+127.0.0.1 d2bgg7rjywcwsy.cloudfront.net
+127.0.0.1 d2wy8f7a9ursnm.cloudfront.net
+127.0.0.1 d3anogn3pbtk4v.cloudfront.net
+127.0.0.1 d3oltyb66oj2v8.cloudfront.net
+127.0.0.1 d5i9o0tpq9sa1.cloudfront.net
+127.0.0.1 dnn506yrbagrg.cloudfront.net
+127.0.0.1 dsms0mj1bbhn4.cloudfront.net
+127.0.0.1 dy48bnzanqw0v.cloudfront.net
+
+# [EFF Tracker Detection / Panopticlick]
+127.0.0.1 do-not-tracker.org
+127.0.0.1 eviltracker.net
+127.0.0.1 trackersimulator.org
+
+# [AdMob by Google www.google.com/admob]
+127.0.0.1 a.admob.com
+127.0.0.1 analytics.admob.com
+127.0.0.1 c.admob.com
+127.0.0.1 media.admob.com
+127.0.0.1 p.admob.com
+
+# [Crashlytics Analytics "Crashlytics is part of Fabric, which has been acquired by Google." try.crashlytics.com]
+127.0.0.1 e.crashlytics.com
+127.0.0.1 reports.crashlytics.com
+127.0.0.1 settings.crashlytics.com
+
+# [DoubleClick by Google www.doubleclickbygoogle.com]
+127.0.0.1 ad.doubleclick.net
+127.0.0.1 ad-g.doubleclick.net
+127.0.0.1 googleads.g.doubleclick.net
+127.0.0.1 ad.mo.doubleclick.net
+127.0.0.1 doubleclick.net
+
+# [other Google analytics and ad services]
+127.0.0.1 www.googleadservices.com
+127.0.0.1 www.google-analytics.com
+127.0.0.1 google-analytics.com
+127.0.0.1 youtubei.googleapis.com
+127.0.0.1 pagead.googlesyndication.com
+127.0.0.1 pagead1.googlesyndication.com
+127.0.0.1 pagead2.googlesyndication.com
+127.0.0.1 googlesyndication.com
+127.0.0.1 fonts.gstatic.com
+
+# [Fiksu mobile advertising]
```

Comment:
Lastpass has this, segment.com, and a few others.  For a password manager I really don't like that they could be mining a lot considering their app asks for app usage access, accessibility svcs, etc.